test/e2e/specs/editor/various/post-editor-template-mode.spec.js

```diff
@@ -271,7 +271,7 @@ class PostEditorTemplateMode {
 		await this.pageUtils.openDocumentSettingsSidebar();
 
 		const templatePanelButton = this.editorSettingsSidebar.locator(
-			'role=button[name="Template"i]'
+			'role=button[name=/Template.*/i]'
```

Comment:
I guess this should better represent to assertion we want to make here :)

```suggestion
			'role=button[name=/^Template/i]'
```